### PR TITLE
Rewriting R code in C++

### PR DESCRIPTION
**Pull request by eliocamp**

### CODE REVIEW

**Comment by eliocamp** [comment]
@hadley in line 644 there's a comment about the user of "sugar". I'm not sure what that means exactly.

File: Rcpp.Rmd [hunk 1]
```diff
@@ -641,6 +641,8 @@ LogicalVector is_naC(NumericVector x) {
 is_naC(c(NA, 5.4, 3.2, NA))
 ```
 
+<!-- Is not clear what "sugar function" means. -->
```
Comment (hadley) [inline Rcpp.Rmd]: Oh I think this whole bit can be removed since I removed the large section on sugar functions.

Comment (eliocamp) [reply Rcpp.Rmd]: I'll change it to 

``Another alternative is the function `is_na()`, which takes a vector and returns a logical vector.``

Comment (hadley) [reply Rcpp.Rmd]: Sorry, I meant remove the entire section that talks about `is_na()`

Comment (eliocamp) [reply Rcpp.Rmd]: Oh, ok. I've removed everything after 
````
```{r}
str(missing_sampler())
```
````

**Pull request merged**